doc/config/_default/config.toml

```diff
@@ -43,6 +43,6 @@ pygmentsUseClasses = true
     weight = 2
 
   [[menu.buttons]]
-    name = "Get The Things Stack"
+    name = "Get Started"
```

Comment:
This is confusing. We have a Getting Started section in the docs and it's also in the header menu.